product/reports/670_Performance by Asset Type - Middleware Servers/170_Web Metrics - every minute over last hour.yaml

```diff
@@ -0,0 +1,59 @@
+---
+where_clause:
+dims:
+reserved:
+title: "EAP Web Metrics - every minute over the last hour"
```

Comment:
I'm wondering if this is a _detailed_ report. The agent is, [by default](https://github.com/hawkular/hawkular-agent/blob/master/hawkular-javaagent-wildfly-feature-pack/src/main/resources/featurepack/content/standalone/configuration/hawkular-javaagent-config.yaml#L93), configured to make measurements each minute or more (depends on the metric). Unless the end user changes the agent config, this report won't be doing any aggregation, but showing every single datapoint (or, probably, repeating the same measurement on more than one row).

The [Request Count](https://github.com/hawkular/hawkular-agent/blob/master/hawkular-javaagent-wildfly-feature-pack/src/main/resources/featurepack/content/standalone/configuration/hawkular-javaagent-config.yaml#L172) is collected once every 5 minutes, for example.

Reply:
You are right, this is not an aggregation report even though it gives meaningful information to the admin, from my point of view.

Regarding the Request Count metrics, it is taken from [here](https://github.com/hawkular/hawkular-agent/blob/master/hawkular-javaagent-wildfly-feature-pack/src/main/resources/featurepack/content/standalone/configuration/hawkular-javaagent-config.yaml#L120) and is collected every 1 minute.